### DISCUSSION TITLE
Whether expr supports raw string?

### DISCUSSION THREAD

**Discussion by zhuliquan** [antonmedv/expr#379] [Q&A]
Whether expr supports raw string? like `\n` in golang or r"\n" in python can ignore '\'.

**Comment by antonmedv** [comment]
Raw string are not supported right now.

**Reply by zhuliquan** [reply]
There are any plan for raw string？

**Reply by antonmedv** [reply]
Yes. With ` char.

**Reply by zhuliquan** [reply]
Whether this is the [implementation](https://github.com/antonmedv/expr/pull/386)
